tests/JoinSupportingQueryBuilderParserTest.php

```diff
@@ -328,4 +343,17 @@ public function testDateNotBetween()
         $this->assertEquals(22, $bindings[0]->day);
         $this->assertEquals(28, $bindings[1]->day);
     }
+
+	public function testJoinNotExistsInThrough()
```

Comment:
Also, with this project, we are trying to stick to `PSR-2` styling (https://www.php-fig.org/psr/psr-2/). Could you please ensure that you are using spaces and not tabs to format code?